### PR TITLE
[opentracing] Fixing context provider imports for scope manager

### PR DESCRIPTION
**Pull request by Maximilien-R**
The goal is to import properly all `context_provider`s to fix the import error when initializing a `ddtrace.opentracer.Tracer` object with a custom scope manager (as explained [here](https://github.com/DataDog/dd-trace-py/issues/770)).

### CODE REVIEW

**Comment by brettlangdon** [comment]
Ha! You beat me to a PR, I basically had the exact same diff

File: ddtrace/opentracer/utils.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-import ddtrace
+import ddtrace.contrib.asyncio
```
Comment (brettlangdon) [inline ddtrace/opentracer/utils.py]:
```suggestion
# DEV: If `asyncio` or `gevent` are unavailable we do not throw an error,
#    `context_provider` will just not be set and we'll get an `AttributeError` instead
import ddtrace.contrib.asyncio
```

This was the only difference I had from what you have here.

Comment (Maximilien-R) [reply ddtrace/opentracer/utils.py]: Do you want me to add this comment to my PR or do you prefer to close my PR in favor of yours?

**Comment by Maximilien-R** [comment]
@Kyle-Verhoog The fact is that the `get_context_provider_for_scope_manager` function already have unit tests (both [`Gevent`](https://github.com/DataDog/dd-trace-py/blob/master/tests/opentracer/test_tracer_gevent.py#L202) & [`Asyncio`](https://github.com/DataDog/dd-trace-py/blob/master/tests/opentracer/test_tracer_asyncio.py#L175)). The tests don't fail certainly because before their execution there is an "explicit" import of their `context_provider` which put them in the python import cache.

File: ddtrace/opentracer/utils.py (hunk 1 above)
Comment (brettlangdon) [reply ddtrace/opentracer/utils.py]: Your call on whether you think this comment is necessary. I am happy to accept either way.

**Comment by brettlangdon** [comment]
@Maximilien-R thank you for this contribution, we just released version `0.19.0` which includes this fix.

**Pull request merged**